.github/workflows/npm_publish.yml

```
@@ -0,0 +1,21 @@
+# This workflow will do a clean install of node dependencies, build the source code and publish the code to the web-serial-polyfill npm package
+# For more information see: https://github.com/marketplace/actions/npm-publish
+
+name: NPM publish
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

We probably need a "setup Node" step here:

```suggestion
    - uses: actions/checkout@v2
    - uses: actions/setup-node@v1
        with:
          node-version: 10
```